### PR TITLE
Change active and exact-active behavior for router-link

### PR DESCRIPTION
**Pull request by posva**
[Rendered](https://github.com/posva/rfcs/blob/router/active-matching/active-rfcs/0000-router-active-link.md)

Make active matching more consistent by avoiding existing caveats by design

- Is this breaking change bringing more value than the old behavior?
- Are any of the alternatives listed more desirable?
- Do you need the `exact` prop?

Please, vote with :+1: and :-1: if this is something you want or not

### CODE REVIEW

File: active-rfcs/0000-router-active-link.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      // empty child
+      { path: '' },
+      // child with id
+      { path: 'child/:id' }
+    ]
+  }
+]
+```
+
+If the current route is `/parent/1/child2`, these links will be active:
```
Comment (cawa-93) [inline active-rfcs/0000-router-active-link.md]: Maybe `/parent/1/child/2` ?

Comment (posva) [reply active-rfcs/0000-router-active-link.md]: fixed!

**Comment by posva** [comment]
I added a few examples about the breaking change that should make this RFC clearer

**Comment by posva** [comment]
I added a section about aliases to explain how they behave with the active behavior

**Comment by posva** [comment]
Added a section about nested absolute aliases

**Comment by posva** [comment]
This RFC is now in final comments stage. An RFC in final comments stage means that:

The core team has reviewed the feedback and reached consensus about the general direction of the RFC and believe that this RFC is a worthwhile addition to the framework.
Final comments stage does not mean the RFC's design details are final - we may still tweak the details as we implement it and discover new technical insights or constraints. It may even be further adjusted based on user feedback after it lands in an alpha/beta release.
If no major objections with solid supporting arguments have been presented after a week, the RFC will be merged and become an active RFC.

**Pull request merged**